modules/common/services/desktop.nix

```diff
@@ -185,7 +185,7 @@ in
             {
               name = "Audio Control";
               description = "System Audio Control";
-              path = "${pkgs.ghaf-audio-control}/bin/GhafAudioControlStandalone --pulseaudio_server=audio-vm:4713";
+              path = "${pkgs.ghaf-audio-control}/bin/GhafAudioControlStandalone --pulseaudio_server=audio-vm:4713 --indicator_icon_path ${pkgs.icon-pack}/preferences-sound.svg";
```

Comment:
Would it be possible to use Gtk.StatusIcon() in Ghaf Audio Control so that the icon is fetched from the current icon theme automagically instead of providing the icon path as an argument here? (https://docs.gtk.org/gtk3/class.StatusIcon.html - admittedly deprecated, but seems to be commonly in use, e.g. blueman-applet)

This came to mind due to https://github.com/tiiuae/ghaf/pull/888, where icon-pack package is no longer used.

Reply:
Thanks for the commend and for the possible solution.

I cannot change AppIndicator to Gtk::StatusIcon as:
1. It is deleted in GTK 4, which Dmitriy is using in his Control Panel;
2. Gtk::StatusIcon doesn't support Wayland;

I will use Gtk::IconTheme to fetch the icon from the theme